File: cpp/lv1/num_has_interval.cpp
// x만큼 간격이 있는 n개의 숫자

#include <string>
#include <vector>

using namespace std;

/* Answer 1.

vector<long long> solution(int x, int n) {
    vector<long long> answer;
    int mul = 0;
    
    for(int i = 1; i < n + 1; i++)
    {
        answer.push_back(x * i);
    }
    return answer;
} */


/* Answer 2.
  
vector<long long> solution(int x, int n) {
    vector<long long> answer;
    int mul = 0;
    
    for(int i = 0; i < n; i++)
    {
        answer.push_back(x * (i + 1)); //(x + (x * i))도 가능
    }
    
    return answer;
} */

File: python/lv1/num_has_interval.py
# x만큼 간격이 있는 n개의 숫자

'''
Answer 1.

def solution(x, n):
    answer = []
    
    for i in range(0, n):
        answer.append(x * (i + 1))
    
    return answer
'''

'''
Answer 2.

def solution(x, n):
    return [x * (i + 1) for i in range(n)]
'''

